services/alchemy_language/v1.js

```diff
@@ -25,77 +28,114 @@ var FEATURES = {
   'concept': 'concepts',
   'relation': 'relations',
   'pub-date': 'publicationDate',
-  'doc-sentiment': 'docSentiment'
+  'doc-sentiment': 'docSentiment',
+  'doc-emotion': 'docEmotions'
 };
 
 module.exports = function (RED) {
-  var cfenv = require('cfenv'),
-    AlchemyAPI = require('alchemy-api');
 
-  var services = cfenv.getAppEnv().services,
-    service;
+  var cfenv = require('cfenv');
+  var watson = require('watson-developer-cloud');
+
+  // Require the Cloud Foundry Module to pull credentials from bound service 
+  // If they are found then the api key is stored in the variable s_apikey. 
+  //
+  // This separation between s_apikey and apikey is to allow 
+  // the end user to modify the key  redentials when the service is not bound.
+  // Otherwise, once set apikey is never reset, resulting in a frustrated
+  // user who, when he errenously enters bad credentials, can't figure out why
+  // the edited ones are not being taken.
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['services' is defined but never used.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2557150214/issues/source?bid=3225115&fileBranchId=3292252#l49)